assets/javascripts/discourse/initializers/discourse-reactions.gjs

```diff
@@ -128,21 +128,26 @@ function initializeDiscourseReactions(api) {
 
         get label() {
           const count = this.notification.data.count;
-          const username = this.username;
+          const nameOrUsername = this.siteSettings.prioritize_full_name_in_ux
+            ? this.notification.data.display_name ||
+              this.notification.data.username
+            : this.username;
```

Comment:
what's the diff between `this.notification.data.username` and `this.username`? and should both be used there? (depending on the site setting? 🤔)

Reply:
`this.username` is a shorthand for `formatUsername(this.notification.data.username)` defined in `NotificationTypeBase`.

Reply:
turns out this was an error on my part, because on the notif object it's referred to as `display_username`. There isn't a `.username` on the notification object.